### PR TITLE
Minor Fixes

### PR DESCRIPTION
**Pull request by draknyte1**
Version bump.
Hopefully fixed HEE Compendium opening when key is unbound. (Fn+F keys open it randomly for me)
Removed AT and replaced with quick reflection, will now build easier.

### CODE REVIEW

**Comment by Dream-Master** [comment]
[crash-2021-08-24_21.34.53-client.txt](https://github.com/GTNewHorizons/Hardcore-Ender-Expansion/files/7041897/crash-2021-08-24_21.34.53-client.txt)

File: src/main/java/chylex/hee/api/message/element/ListValue.java [hunk 1]
```diff
@@ -31,17 +32,26 @@ public boolean checkType(NBTBase tag){
 
 	@Override
 	public boolean checkValue(NBTBase tag){
-		for(NBTBase element:(List<NBTBase>)((NBTTagList)tag).tagList){
-			if (!condition.checkType(element) || !condition.checkValue(element))return false;
-		}
-		
+		List tagList = ReflectionUtils.getFieldValue((List<NBTBase>)((NBTTagList)tag), "tagList");
```
Comment (Glease) [inline src/main/java/chylex/hee/api/message/element/ListValue.java:35]: There is no reason to use reflection here either way. The interface for iterating over NBTTagList content is quite cumbersome, but it does exist. Might use that instead.

File: src/main/java/chylex/hee/mechanics/compendium/events/CompendiumEventsClient.java [hunk 2]
```diff
@@ -114,12 +120,19 @@ private void displayAchievement(Achievement achievement){
...
+		return true;
+	}
+	
 	@SubscribeEvent
 	public void onPlayerLogin(PlayerLoggedInEvent e){
 		Stopwatch.time("CompendiumEventsClient - key conflict check");
 		
 		for(KeyBinding kb:Minecraft.getMinecraft().gameSettings.keyBindings){
-			if (kb != instance.keyOpenCompendium && kb.getKeyCode() == instance.keyOpenCompendium.getKeyCode()){
+			if (isKeybindingValid() && kb != instance.keyOpenCompendium && kb.getKeyCode() == instance.keyOpenCompendium.getKeyCode()){
```
Comment (Glease) [inline src/main/java/chylex/hee/mechanics/compendium/events/CompendiumEventsClient.java:135]: Probably could have moved this out of for loop.

File: src/main/java/chylex/hee/api/message/element/ListValue.java [hunk 3]
```diff
@@ -31,17 +32,26 @@ public boolean checkType(NBTBase tag){
...
+			}
+		}		
 		return true;
 	}
 
 	@Override
 	public List<T> getValue(NBTBase tag){
 		List<T> elements = new ArrayList<>();
-		for(NBTBase element:(List<NBTBase>)((NBTTagList)tag).tagList)elements.add(condition.getValue(element));
+		List tagList = ReflectionUtils.getFieldValue((List<NBTBase>)((NBTTagList)tag), "tagList");
```
Comment (Glease) [inline src/main/java/chylex/hee/api/message/element/ListValue.java:49]: Same here.

**Comment by Dream-Master** [comment]
@draknyte1

**Comment by Dream-Master** [comment]
@Glease

**Review by Glease — APPROVED** [review]

**Pull request merged**